src/connectors/system.c

```diff
@@ -27,6 +27,7 @@
 #include "local.h"
 #include "sample.h"
 #include "connectors/common.h"
+#include <sys/unistd.h>
```

Comment:
The correct header for `unlink` is just `<unistd.h>`

https://pubs.opengroup.org/onlinepubs/9699919799/functions/unlink.html

```suggestion
#include <unistd.h>
```

Reply:
If the intention is the Gtk `g_unlink` function (which works on Windows as well as POSIX) then I think the heaer is `<glib/gstdio.h>`

Reply:
Thanks, @jwakely.

I've tried to solve this issue a couple of times without success.

I confirm that adding the `<glib/gstdio.h>` header solves the warning while using `g_unlink`, which makes the code portable.

@snd1, thanks for the PR but we need to change it.